### PR TITLE
Fix missing documentation warnings

### PR DESCRIPTION
**Pull request by KarloX2**

### CODE REVIEW

File: SharpSnmpLib/Pipeline/SetMessageHandler.cs [hunk 1]
```diff
@@ -85,9 +85,10 @@ public void Handle(ISnmpContext context, ObjectStore store)
                     {
                         status = ErrorCode.NoAccess;
                     }
-                    catch (ArgumentException)
+                    catch (ArgumentException ex)
                     {
-                        status = ErrorCode.WrongType;
+                        if (!Enum.TryParse<ErrorCode>(ex.Message, out status) || status == ErrorCode.NoError)
```
Comment (lextm) [inline SharpSnmpLib/Pipeline/SetMessageHandler.cs]: This is more than "fix missing documentation warnings". Can you move it to another pull request for review?

Comment (KarloX2) [reply SharpSnmpLib/Pipeline/SetMessageHandler.cs]: I already tried but I don't known how (have not too much experience with GitHub). Can you tell me? I first created the "Fix missing documentation warnings" PR, then added the "More specific error code in set response" commit to my fork and that just showed up in the original PR without further action from my side. How can I create two separate PRs?

Comment (lextm) [reply SharpSnmpLib/Pipeline/SetMessageHandler.cs]: GitHub has its documentation https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/

Comment (KarloX2) [reply SharpSnmpLib/Pipeline/SetMessageHandler.cs]: This is quite complicated stuff, I'll have to spend some time to learn more about how GitHub works, but currently I don't have the time. So I just reverted the extra commit. Once you accept my PR I will create a new fork and create the other PR, ok?
